content/python/concepts/strings/terms/endswith/endswith.md

```diff
@@ -0,0 +1,72 @@
+---
+Title: '.endswith()'
+Description: 'Checks whether or not a string ends with the given value.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+TThe `.endswith()` method returns `True` if a particular value is attached to the end of a given string. Otherwise, it returns `False`.
```

Comment:
```suggestion
The `.endswith()` method checks a value against a given string and returns `True` if the value exists at the end of that string. Otherwise, it returns `False`.
```